.circleci/config.yml

```diff
@@ -322,7 +322,7 @@ jobs:
             
             shopt -s globstar
             
-            SPECS=$(circleci tests glob "e2e_tests/features/**/*.feature" | circleci tests split --split-by=timings --show-counts)
+            SPECS=$(circleci tests glob "e2e_tests/features/*.feature" | circleci tests split --split-by=timings --show-counts)
```

Comment:
Doesn't this mean that it won't run the tests in the features/e2e folder? Or any other folder that we choose to create in the future?

Reply:
The tests under features/e2e folder are currently skipped for e2e_local_test job. But even to skip these tests cypress spends a few mins on the pipeline to execute it and skip it. By excluding the features/e2e folder we should save those few mins cypress takes, hopefully.

Reply:
I see.  That seems to be somewhat hidden and not obvious that it is happening.  Can we make it more explicit with appropriate folder hierarchy and naming?

Reply:
updated as requested..